chromadb/telemetry/posthog.py

```diff
@@ -49,7 +49,7 @@ def _direct_capture(self, event: TelemetryEvent) -> None:
             posthog.capture(
                 self.user_id,
                 event.name,
-                {**(event.properties), "chroma_context": self.context},
```

Comment:
why are we flattening this?

Reply:
You can query these with posthog SQL easily - see https://app.posthog.com/insights/pliuCZsS

Reply:
Slightly lower cognitive overhead to attach tags to events in a uniform way. I'm not attached to this, though, happy to revert. WDYT?

Reply:
My only concern is data continuity.

Reply:
I think we can afford some discontinuity right now, since we won't actually lose the data, and I would prefer we had the simpler interface. I think this is OK.

Reply:
Bikeshedding. I don't care.

Reply:
Let the record show that the the Chroma team resolved to paint the bike shed red.

Reply:
no blue